.yamato/project-tests.yml

```diff
@@ -41,7 +41,15 @@ test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
     flavor: {{ platform.flavor}}
   commands:
     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-    - {% if platform.name == "centos" %}DISPLAY=:0 {% endif %}upm-ci project test -u {{ editor }} --project-path {{ project.path }} --type project-tests
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - unity-downloader-cli -u {{ editor }} -c editor -w --fast
+    - |
+{% if platform.name == "ubuntu" %}      export DISPLAY=:0{% endif %}
+      {{ platform.editorpath }} -projectpath testproject -batchmode -nographics -quit -logfile BuildMultiprocessTestPlayer.log -executeMethod Unity.Netcode.MultiprocessRuntimeTests.BuildMultiprocessTestPlayer.BuildDebug
+{% if platform.name == "mac" %}    -  sudo codesign --force --deep --sign - ./testproject/Builds/MultiprocessTests/MultiprocessTestPlayer.app{% endif %}
+    - |
+{% if platform.name == "ubuntu" %}      export DISPLAY=:0{% endif %}
```

Comment:
Not sure you have to export twice - you do it on the previous command... or else you could follow the non-export pattern that was here previously - might be a bit easier to read what's going on instead of these multi-line executions

Reply:
Oh, right.

Reply:
done